.gitignore

```diff
@@ -9,6 +10,7 @@ __pycache__/
 # Environments
 .env
 .venv
+.idea
```

Comment:
What is this?

Reply:
config files for intellij IDE

Reply:
Are you using intelliJ for python script?